ts/@types/react-native-screen-brightness.d.ts

```diff
@@ -0,0 +1,9 @@
+declare module 'react-native-screen-brightness'{
+    const ScreenBrightness: {
+        getAppBrightness(): Promise<number>;
+        setAppBrightness(val: number): void;
```

Comment:
Looking the source could it seems to be a promise too
[Android](https://github.com/robinpowered/react-native-screen-brightness/blob/634124c3bb775e084946645ace02154e3f0414eb/android/src/main/java/com/robinpowered/react/ScreenBrightness/ScreenBrightnessModule.java#L232)
[iOS](https://github.com/robinpowered/react-native-screen-brightness/blob/634124c3bb775e084946645ace02154e3f0414eb/ios/ScreenBrightness/ScreenBrightness.m#L32)